lib/Utils/Systemd.pm

```diff
@@ -48,10 +48,13 @@ sub disable_and_stop_service {
     $args{mask_service}   //= 0;
     $args{ignore_failure} //= 0;
 
-    systemctl("mask $service_name", ignore_failure => $args{ignore_failure}) if ($args{mask_service});
-    systemctl("disable $service_name", ignore_failure => $args{ignore_failure}) unless ($args{mask_service});
```

Comment:
This is wrong. Someone has changed it. Disable should happen always.

Reply:
It seems there is a bug in older versions: https://bugzilla.suse.com/show_bug.cgi?id=1135080